storage: separate samples into standalone files pt2 (generated by scr…

IlyaFaer opened the pull request:
…ipt)

Made by script. Every sample moved into standalone file. Names of files writen by samples names (as docs require). Imports are made with `goimports -w .`. Start and end tags moved to starts and ends of files

`objects` directory

Commit:
storage: separate samples into standalone files pt2 (generated by script)

IlyaFaer commented:
@AlisskaPie

edorivai committed:
iot: update inconsistent comment on JWT algorithm (#1165)

Commit:
deleted samples duplicates, renamed package, fixed imports

Commit:
add client init and comments into every samples

Commit:
cosmetic changes

Commit:
del excess comments

Commit:
formating and fixing according to CONTRIBUTING.md

renovate-bot committed:
all: update all (#1166)

grayside committed:
run/logging-manual: strip timestamp prefix from log instead of test (#1134)

* run/logging-manual: strip timestamp prefix from log instead of test

* run/logging-manual: move log.SetFlags to an init function

* update comment to explain timestamp prefix is bad

Commit:
add doc.go files into packages, set license year to 2020, some minor changes

Commit:
add client closing

Commit:
add writers

grayside committed:
run/grpc-ping: update README beta commands to GA (#1169)

AlisskaPie commented on storage/acl/bucketListACL.go:
```diff
@@ -0,0 +1,45 @@
+	"cloud.google.com/go/storage"
+)
+
+// bucketListACL lists bucket ACL.
```
I think it'll be better to swap these words.
`ACL bucket`

AlisskaPie commented on storage/acl/bucketListACLFiltered.go:
```diff
@@ -0,0 +1,48 @@
+	"cloud.google.com/go/storage"
+)
+
+// bucketListACLFiltered lists bucket ACL using a filter.
```
`ACL bucket`

AlisskaPie commented on storage/acl/deleteObjectACL.go:
```diff
@@ -0,0 +1,41 @@
+	"cloud.google.com/go/storage"
+)
+
+// deleteObjectACL removes default ACL from object.
```
Add an/the with object
`from the object`

IlyaFaer replied:
Done

AlisskaPie commented on storage/acl/doc.go:
```diff
@@ -0,0 +1,17 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package acl contains samples for tweaking ACLs for
```
I'm not sure that tweaking is the right word here
WDYT about _ACLs configuration_?

IlyaFaer replied:
Done

AlisskaPie commented on storage/objects/generateV4GetObjectSignedURL.go:
```diff
@@ -0,0 +1,59 @@
+	"golang.org/x/oauth2/google"
+)
+
+// generateV4GetObjectSignedURL generates object signed URL with GET method.
```
Maybe write it this way?
`object-signed`

AlisskaPie commented on storage/objects/listByPrefix.go:
```diff
@@ -0,0 +1,70 @@
+	if err != nil {
+		return err
+	}
+    defer client.Close()
```
Try `gofmt`, it looks like here must be a space before `defer client.Close()`

AlisskaPie commented on storage/objects/listByPrefix.go:
```diff
@@ -0,0 +1,70 @@
+	}
+    defer client.Close()
+	// Prefixes and delimiters can be used to emulate directory listings.
+	// Prefixes can be used filter objects starting with prefix.
```
Need to add "to"
`used to filter`

IlyaFaer replied:
Done

AlisskaPie commented on storage/objects/listByPrefix.go:
```diff
@@ -0,0 +1,70 @@
+	// Prefixes and delimiters can be used to emulate directory listings.
+	// Prefixes can be used filter objects starting with prefix.
+	// The delimiter argument can be used to restrict the results to only the
+	// objects in the given "directory". Without the delimiter, the entire  tree
```
`entire  tree`
Need to delete extra space

AlisskaPie commented on storage/objects/objects_test.go:
```diff
@@ -257,7 +240,7 @@ func TestV4SignedURL(t *testing.T) {
 		t.Errorf("httpClient.Do: %v", err)
 	}
 	getBuf := new(bytes.Buffer)
```
Maybe it'll be better to keep one style of defining buffer?
`var buf bytes.Buffer`

AlisskaPie commented on storage/objects/releaseEventBasedHold.go:
```diff
@@ -0,0 +1,44 @@
+	"cloud.google.com/go/storage"
+)
+
+// releaseEventBasedHold releases object with event-based hold.
```
` releases an object `

AlisskaPie commented on storage/objects/releaseTemporaryHold.go:
```diff
@@ -0,0 +1,44 @@
+	"cloud.google.com/go/storage"
+)
+
+// releaseTemporaryHold releases object with temporary hold.
```
`releases an object`

AlisskaPie commented on storage/objects/rotateEncryptionKey.go:
```diff
@@ -0,0 +1,45 @@
+	defer client.Close()
+
+	obj := client.Bucket(bucket).Object(object)
+	// obj is encrypted with key, we are encrypting it with the newKey.
```
`with the key`

AlisskaPie commented on storage/objects/rotateEncryptionKey.go:
```diff
@@ -0,0 +1,45 @@
+	"cloud.google.com/go/storage"
+)
+
+// rotateEncryptionKey encrypts an object with new key.
```
I would add "a"
`with a new key`

AlisskaPie commented on storage/objects/signedURL.go:
```diff
@@ -0,0 +1,43 @@
+
+	"cloud.google.com/go/storage"
+)
+
+func signedURL(bucket, object string) error {
```
Need to add a comment

AlisskaPie commented on storage/objects/signedURL.go:
```diff
@@ -0,0 +1,43 @@
+	// Download a p12 service account private key from the Google Developers Console.
+	// And convert it to PEM by running the command below:
+	//	$ openssl pkcs12 -in key.p12 -passin pass:notasecret -out my-private-key.pem -nodes
+	pkey, err := ioutil.ReadFile("my-private-key.pem")
```
Forgot to add comments with bucket-object names examples inside the function body

AlisskaPie commented on storage/objects/signedURL.go:
```diff
@@ -0,0 +1,43 @@
+	if err != nil {
+		return err
+	}
+	fmt.Println(url)
```
Need to rewrite it with io.Writer

AlisskaPie commented on storage/objects/writeWithKMSKey.go:
```diff
@@ -0,0 +1,48 @@
+)
+
+// writeWithKMSKey writes an object using Cloud KMS encryption.
+func writeWithKMSKey(bucket, object string, keyName string) error {
```
Type can be collapsed into one word

AlisskaPie commented on storage/objects/writeWithKMSKey.go:
```diff
@@ -0,0 +1,48 @@
+	"cloud.google.com/go/storage"
+)
+
+// writeWithKMSKey writes an object using Cloud KMS encryption.
```
Maybe add `for the bucket` and `using the Cloud KMS encryption`?

Commit:
cosmetic changes

grayside committed:
jobs: broken quickstart link (#1172)

renovate-bot committed:
all: update all (#1174)

grayside committed:
testing: go vet submodule compatibility (#1168)

* testing: go vet submodule compatibility

* memorystore: remove unneeded go.mod and go.sum

* run/hello-broken: fix broken test

* testing/system_tests.sh: bash style correction

grayside committed:
cloudsql/mysql, testing: add support for Cloud Run, fix vet for subdirs (#1151)

tritone committed:
storage: add timeouts to samples (#1173)

The storage client doesn't automatically include timeouts by default,
and users are expected to set their own. This should be included
in sample code.

sethvargo committed:
secretmanager: add IAM samples (#1177)

deepnirmal committed:
appengine: added closure of client to avoid memory spike (#1183)

bradmiro committed:
dataproc: add usage text to sample comment (#1182)

renovate-bot committed:
all: update all (#1209)

renovate-bot committed:
all: update golang Docker tag to v1.13 (#1210)

Commit:
add more descriptive errors, empty line, sort imports, erase MainTests

Commit:
fix some sample tags and file names

Commit:
revert file deletion

grayside committed:
testing: execute all tests in go submodules (#1208)

* testing: update flow, optimize static analysis, run fmt in submodules

* testing: remove duplicate scan execution and added comments

* testing: align test variable name change

* testing: feedback/clarity improvements

* testing: add go submodule to go test

* remove changes added in test testing

* testing: fix hidden test failure in functions/ocr/app

Commit:
small fixes

tbpg committed:
appengine: remove gophers sample (#1181)

It is no longer used.

tbpg committed:
storage: skip HMAC S3 test (#1214)

tbpg committed:
all: go mod tidy

tbpg committed:
run: don't use log.Writer(), added in Go 1.13

Commit:
Merge branch 'master' of https://github.com/GoogleCloudPlatform/golang-samples into separate_storage_samples-pt2

renovate-bot committed:
all: update all (#1222)

Commit:
improve error messages

Commit:
rename ACL tags and files

Commit:
Merge branch 'separate_storage_samples-pt2' of https://github.com/q-logic/golang-samples into separate_storage_samples-pt2

Commit:
discard unused buffers

Commit:
small fixes to make tests OK

Commit:
write new object directly

tbpg committed:
Merge branch 'master' into separate_storage_samples-pt2

shollyman committed:
bigquery: add defers to explicitly close all clients (#1219)

Our samples implicitly leveraged short lifetimes, and thus didn't
include explicit closures.  We've seen examples in the wild of
people missing this for longer-lived clients, so this makes all samples
more explicit about client lifecycle.

tbpg committed:
bigquery: run gofmt -w -s (#1223)

MAJA-Lin committed:
run: fix typo in samples (#1230)

shollyman committed:
bigquery: update streaming insert snippet (#1231)

This snippet now demonstrates how to disable best effort deduplication,
which allows for higher quota/throughput in streaming insert usage.

tbpg committed:
testing: only check go mod tidy with vet version (#1234)

grayside committed:
run/hello-broken: add e2e testing (#1226)

tbpg committed:
testing: call Trampoline cleanup script (#1236)

tbpg committed:
testing/kokoro: use buildcop.sh script (#1237)

tbpg committed:
functions/slack: remove token references (#1238)

tbpg committed:
spanner: skip flaky TestSample test (#1239)

tbpg committed:
testing/kokoro: remove buildcop.sh script, use binary (#1240)

See https://github.com/googleapis/repo-automation-bots/pull/310.

tbpg committed:
testing: chmod +x buildcop binary (#1243)

hengfengli committed:
spanner: fix recurring session deletion error (#1241)

tdh911 committed:
securitycenter: add apiv1p1beta1 notification snippets (#1228)

hengfengli committed:
spanner: add a sleep time to fix the flaky test (#1242)

sirtorry committed:
speech: add speech adaptation sample (#1229)

sirtorry committed:
speech: fix syntax errors (#1246)

tbpg committed:
securitycenter, speech: gofmt -w -s (#1248)

tbpg committed:
speech: run goimports (#1253)

grayside committed:
run/markdown-preview: sample for a secure 2-tier app (#1179)

* run/markdown-preview: initial commit

* run/markdown-preview/editor: go fmt

* tmp

* run/markdown-preview: revise structure and fix unit tests

* run/markdown-preview: add license headers to editor

* run/markdown-preview: remove extraneous region tags

* markdown-preview: rename render -> renderer and expand test coverage

* markdown-preview: remove go113 error handling

* markdown-preview: update go.mod

* markdown-preview: fix typo

* markdown-preview: more error handling

* markdown-preview: fixing vet flags

* markdown-preview: refine markdown text

* markdown-preview: golang-practices feedback

* markdown-preview: update README

* run/markdown-preview: remove quotation

* run/markdown-preview: add doctype

* run/markdown-preview: add whitespace to default markdown

Commit:
add timeouts into new samples

Commit:
add prints

tbpg committed:
testing: output separate sponge_log.xml and sponge_log.log files (#1254)

tbpg committed:
testing: add Go 1.14 (#1255)

tbpg committed:
Merge branch 'master' into separate_storage_samples-pt2
